Scroll_Everywhere/scrolle.user.js

```diff
@@ -134,3 +138,12 @@ function fFalse() {
 function slowF(x) {
     return 1 / (1 + Math.pow(Math.E, (-0.1 * x)));
 }
+
+function scroll(speed) {
```

Comment:
this is the main function, it has been written in jQuery, to minimize the lines of code to write